File: lesson4/c_c++/homework/homework4_29.c
//: lesson4/c_c++/homework/homework4_29.c
#include <stdio.h>

int main() {
    int numb_a, numb_n;
    printf("Для получения простых чисел,\n");
    printf("Введите первое число => ");
    scanf("%d", &numb_n);
    printf("Введите второе число => ");
    scanf("%d", &numb_a);
    
    while (numb_n < numb_a) {
        int numb_k = 2;
        while (numb_n % numb_k != 0) {
            numb_k++;
        }
        if (numb_k == numb_n) {
            printf("Простое число => %d\n", numb_n);
        }
        numb_n++;
    }
    return 0;
}

File: lesson4/python3/homework/homework4_29.py
#!/usr/bin/python3
#: lesson4/python3/homework/homework4_29.py
# Напишите программу, которая вводит числа a и b и выводи все простые числа в
# диапазоне a и b.

print("Программа выводит все простые числа в диапазоне a and n!")
n = int(input("Введите число => "))
a = int(input("Введите число => "))
print()
if n == 1:
    n += 1

while n < a:
    k = 2
    while n % k != 0:
        k += 1
    print()
    print(k)
    print()
    if k == n:
        print("Простые числа =>", n)
    n += 1

